Dockerfile

```diff
@@ -3,6 +3,18 @@ WORKDIR /go/src/github.com/openshift/cluster-node-tuning-operator
 COPY . .
 RUN make build
 
+FROM centos:7 as tuned
```

Comment:
Just for my own knowledge,  why do we use FROM centos:7 twice here ? (one as tuned and then again?)